banyand/stream/stream_query.go

```diff
@@ -39,8 +41,12 @@ type Query interface {
 }
 
 type Stream interface {
+	io.Closer
+	Write(value *streamv2.ElementValue) error
```

Comment:
Once the write returns an empty error, indices and raw data are all there. @lujiajing1126

Reply:
From your PR's CI failures, the stream tests are flaky. This new write function can fix this defect.